### DISCUSSION TITLE
Want to pre-link a project in a repo... is this bad form?

### DISCUSSION THREAD

**Discussion by droplab** [vercel/next.js#36720] [Help]
Is it bad form to modify the .gitignore to allow .vercel/project.json while blocking everything else in the .vercel folder in order to have a project pre-linked for another team member who is already using the vercel cli ?

**Comment by icyJoseph** [comment]
Most projects I've seen document creating this file as part of the process. I guess, as long as are sure that you are not leaking any sensitive data through that file, it should be ok. 

I think you can even move that file to another location,  and avoid modifying your `.gitignore` altogether, as they do here: https://github.com/anuraghazra/github-readme-stats/blob/master/vercel.json, and then while running the `vercel` command, you can specify where to find the file: 

```
   -A FILE, --local-config=FILE   Path to the local `vercel.json` file
```
